src/makeRobot.js

```diff
@@ -39,6 +40,62 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    coords: {
+      'x': 0,
+      'y': 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value = 1) {
+      if (value >= 1) {
+        this.coords.y += value;
```

Comment:
```suggestion
    goForward(step = 1) {
      if (step >= 1) {
        this.coords.y += step;
```

И аналогично в других методах